### ISSUE TITLE
Harlequin Codex - 2015

### ISSUE THREAD

**Issue by Kangodo** [BSData/wh40k-7th-edition#759]
Seems like we have an entire new Codex to work with!
Is anyone here working on it or can I make a start this evening?

**Comment by Kangodo** [comment]
Starting on profiles then :)

**Comment by Toreador13** [comment]
Hey, I have a file finished from all of the leaked info, it may be a good place to start from. It was complete from all of the slates and profiles we had seen, just no formations. Might be a good starting point.

**Comment by Kangodo** [comment]
Awesome! Could you commit and upload it?

**Comment by Toreador13** [comment]
I'll see if I can from work. I tend to have issues with the firewall when doing that. If I can't I will try it tonight from home.

**Comment by Toreador13** [comment]
Will have to wait until I get home it looks like. But it already has profiles and such. Was messing around with how to do the Artifacts, so like I said it's a work in progress, but a lot is already there. Shouldn't take much more for a person with the codex to finish it.

**Comment by Kangodo** [comment]
No problem, I was bored anyway.. ;)
I'm pretty good with artifacts and such and I think it worked out nicely.
All the stuff you said you did were things I had already finished anyway.

**Comment by Toreador13** [comment]
Not a problem, I uploaded the file I had created. Copy and paste what you want. 
It's Harlequins - Codex(2015).cat

Need to edit the gst for the new Masque also.

**Comment by Kangodo** [comment]
I already did that, so that's not needed anymore.
It also seems like our part is nearly identical, the only difference:
-Relics, I've weaved mine in the original gear with a lot of modifiers.
-Troupe Players are a bit messed up.
-Shadowseer Shuriken Pistol cannot be taken away.
-Voidweavers have the wrong point cost.
-A lot of wargear isn't implemented.

The thing I needed most were the Warlord Traits and Psychich abilities, but you don't have that either.
I think the wisest decision would be to delete the file with the most errors and work together on mine.
